microfactory/microfactory.go

```diff
@@ -69,6 +69,7 @@ var (
 
 	goToolDir = filepath.Join(runtime.GOROOT(), "pkg", "tool", runtime.GOOS+"_"+runtime.GOARCH)
 	goVersion = findGoVersion()
+	isGo18    = strings.Contains(goVersion, "go1.8")
```

Comment:
We'd need to check to see if ReleaseTags does not include go1.9, since it'll include all previous version tags. I'll try that out.